README.md

```diff
@@ -13,3 +14,28 @@
+  
+- The difference in performance that we are sometimes talking about with asymptotic analysis can to be very minuscule between different algorithms. With this in mind, there are many cases where the difference in performance can be functionally the same, within margin of error, rounding based on significant figures of runtime calculation, etc. For example, a specific algorithm has an average case runtime of n^2logn while another has one of n^2. At an input size of 10, which is a reasonable input size for basic testing when coding, the first algorithm would have a runtime of 100 seconds and the second would have the same runtime.
+  
+- Opposite of the other two reasons, depending on the notation used (big/little o, $\theta$, etc) the difference in functions can stray far. In this case, the difference in runtime with a small input size can actually be larger while the difference in runtime with a large input size can be smaller.
+
+2:
+
+log(1000) = 3 seconds
+actual time: 5 seconds
+
+log(10000) * 5/3 = 6.67 seconds
+
+Reasoning: Using the average case runtime complexity for a binary search tree (O(logn)) we can guess that a tree with 1000 elements would have a find function that takes around 3 seconds, however the actual time is 5 seconds. If we create a ratio of actual time to theoretical time, we can guess that the actual runtime based on input would 5/3 of the runtime calculated through asymptotic analysis. Thus, for an input of 10,000 elements I would guess that it should take an average of 6.67 seconds to find a particular element.
+ 
+3:
+
+- I used the average case asymptotic complexity in order to perform the calculations in the previous question, which is not always how the time complexity will work out depending on how the elements are put into the tree in the first place. Say that the elements are sorted backwards relative to the specific element that we are searching the tree for, meaning that the element is theoretically at the bottom of the tree. In this case, we would use the worst case time complexity of O(n).
+
+- The specific element that we are looking for could be very deep into the tree, which would have a large effect on how long it takes to actually find. As an example of this, say that a comparison on a specific computer takes 0.01 seconds to complete. If 100 comparisons are made when the element is found in the tree, it will have taken 1 second to complete the search. However, if the given element is at the bottom of the tree with 10,000 comparisons being made, then it will take 100 seconds to complete the search.
```

Comment:
The height of the tree for the small input is 10 and for the small input 13. So we're not making anything even close to 100 comparisons.